molo/profiles/templates/rangefilter/date_filter.html

```diff
@@ -0,0 +1,13 @@
+<!--
+
+File: molo/core/templates/daterange_filter/filter.html
+
+This file is empty because:
+
+- The django-daterange-filter package is unmaintained and does not support
+  newer versions of Django admin
+- If this template does not exist Django raises an exception
+- Removing the package completely is hard because we use it with our own
+  custom templates in the Wagtail admin
```

Comment:
but you have removed the package? it's no longer part of the setup.py file?

Reply:
We changed the package. Still needs template